### PR TITLE
Remove rancher external ip webhook

### PR DESCRIPTION
**Pull request by KevinJoiner**
Issue: https://github.com/rancher/rancher/issues/38954

# Problem
Rancher `v2.7` Kubernetes support matrix was updated to be `>= v1.22`. Rancher [external-ip-webhook](https://github.com/rancher/externalip-webhook) is unsupported starting at `v1.22`
> Note: This chart is deprecated for kubernetes version 1.21 and unsupported starting with 1.22. To mitigate CVE-2020-8554, enable the [DenyServiceExternalIPs admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips) on the cluster.

# Changes
- ~~Previous version bump to rancher-external-ip-webhook was reverted.~~
- ~~doNotRelease=true was set in the charts.yaml~~
- Previously created charts were removed with `make remove`
- Removed `packages/rancher-external-ip-webhook`

# Additional Info:
Before the support matrix was updated, changes were made to the chart's version, and new charts were generated. These changes have not been released and are only present in the dev-2.7 branch

Version Bump PR: https://github.com/rancher/charts/pull/2078
Charts Generation PR: https://github.com/rancher/charts/pull/2104

### CODE REVIEW

**Review by cbron — DISMISSED** [review]
Last version was `100.0.2+up1.0.1`, so this looks right to me. 
Though we still need an answer on if we should remove all old versions or not in the 2.7 branches.

File: packages/rancher-external-ip-webhook/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 url: https://github.com/rancher/externalip-webhook/releases/download/v1.0.1/rancher-externalip-webhook-1.0.1.tgz
-version: 101.0.0
+version: 100.0.2
 doNotRelease: false
```
Comment (cbron) [inline packages/rancher-external-ip-webhook/package.yaml:3]: @KevinJoiner I talked to Prachi and they want to remove the package.yaml for removed charts.

**Review by cbron — APPROVED** [review]
lgtm, I'd still like someone from team 3 to review if possible

**Review by prachidamle — APPROVED** [review]
LGTM

**Review by pennyscissors — APPROVED** [review]

**Pull request merged**